npm/javy-cli/index.js

```diff
@@ -51,7 +51,7 @@ async function downloadBinary(version) {
 	const targetPath = binaryPath(version);
 	const compressedStream = await new Promise(async (resolve) => {
 		const url = binaryUrl(version);
-		console.log(`Downloading ${NAME} ${version} to ${targetPath}...`);
+		console.log(`Downloading ${NAME} ${version} to ${targetPath}`);
```

Comment:
I removed the `...` because the name of the file ends with a something that looks like `v1.1.1` which is rendered as `v1.1.1...` which makes it harder to figure out where the path ends.